GOVERNANCE.md

```diff
@@ -0,0 +1,153 @@
+# Eiffel Community Governance
+
+Eiffel Community initiated the work to establish governance and put the necessary structures in place in order to have a healthy, sustainable,
```

Comment:
Maybe:
"Governance of the Eiffel Community is structured according to the following document, this structure in intended to provide a healthy, sustainable and attractive community for Eiffel development and collaboration."

Reply:
will amend